docs/modules/operation/pages/admin/introduction.adoc

```diff
@@ -11,3 +11,4 @@ This section describes additional administrative tasks you may want to perform o
 * xref:admin/http-ssl.adoc[HTTPS/SSL configuration]
 * xref:admin/request-logging.adoc[request logging]
 * xref:admin/restart.adoc[shutdown and restart]
+* xref:admin/housekeeping/introduction.adoc[housekeeping tasks]
```

Comment:
I might suggest removing this page and modifying docs/modules/operation/nav.adoc line 155 to be
` * {page-component-title} administration`
That will make it so there is no page and clicking on the nav link item just expands/collapses the group without redirecting the user to a placeholder page. (see https://github.com/OpenNMS/opennms/blame/develop/docs/modules/reference/nav.adoc#L102 Reference:Provisioning for example)